doc/whats_new/v1.4.rst

```diff
@@ -400,13 +400,14 @@ Changelog
 :mod:`sklearn.preprocessing`
 ............................
 
-- |MajorFeature| :class:`preprocessing.MinMaxScaler` and
-  :class:`preprocessing.MaxAbsScaler` now
+- |MajorFeature| :class:`preprocessing.MinMaxScaler`,
+  :class:`preprocessing.MaxAbsScaler` and :class:`preprocessing.Binarizer` now
   support the `Array API <https://data-apis.org/array-api/latest/>`_. Array API
   support is considered experimental and might evolve without being subject to
   our usual rolling deprecation cycle policy. See
   :ref:`array_api` for more details.
-  :pr:`26243` by `Tim Head`_ and :pr:`27110` by :user:`Edoardo Abati <EdAbati>`.
+  :pr:`26243` by `Tim Head`_, :pr:`27110` by :user:`Edoardo Abati <EdAbati>`
+  and :pr:`27588` by :user:`Yaroslav Korobko <Tialo>`
```

Comment:
Let's hear what others think: I am kind of -0 on cramming things into one changelog entry. As a reader of a changelog I tend to look at each bullet point/entry and assume it is about exactly one thing (this means. if the first few words don't sound relevant I skip it).

But maybe I should have made this comment already earlier (say in #27110)

Reply:
I don't like current structure too. Maybe something like this would be more preferable:
```
The following classes now support Array API ...:
* :class: ... :pr: ... :user: ...
* :class: ... :pr: ... :user: ...
etc
```

Reply:
Thoughts @ogrisel ?

I think my preference is one normal changelog entry per class/PR

Reply:
Indeed one bullet point entry by change is probably better (but all grouped under the array API section and the metrics / estimators / meta-estimator subsections).